tests/appsTest.js

```diff
@@ -1,6 +1,6 @@
 describe('Main: Currently testing apps management,', function () {
 // CURRENT TIME
-  var timeRightNow = new Date().getTime()
+  var timeRightNow = Math.random().toString(36).substr(2, 9)
```

Comment:
better use hardcoded values for deterministic results

Reply:
The idea is to add a suffix on each test run so that files and folders don't collide when rerunning the test suite.

Reply:
ah ok